src/Presentation/appsettings.json

```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings" : {
+    "DefaultConnectionString" : "server=localhost;userid=myusername;password=mypassword;database=mydatabase"
+  }
```

Comment:
Should we remove this whole file? Should we use the `Settings/settings.default.json` one instead?